src/X3DCanvas.js

```diff
@@ -1558,24 +1558,27 @@ x3dom.X3DCanvas.prototype.getVRFrameData = function ( xrFrame )
         if ( inputSource.gripSpace )
         {
             const inputPose = xrFrame.getPose( inputSource.gripSpace, this.xrReferenceSpace );
-
-            vrFrameData.controllers[ inputSource.handedness ] = {
-                gamepad : inputSource.gamepad,
-                type    : inputSource.profiles[ 0 ],
-                pose    : {
-                    position : [
-                        inputPose.transform.position.x,
-                        inputPose.transform.position.y,
-                        inputPose.transform.position.z
-                    ],
-                    orientation : [
-                        inputPose.transform.orientation.x,
-                        inputPose.transform.orientation.y,
-                        inputPose.transform.orientation.z,
-                        inputPose.transform.orientation.w
-                    ]
-                }
-            };
+            if ( typeof inputPose === "object" && inputPose !== null &&
```

Comment:
https://immersive-web.github.io/webxr/#dom-xrframe-getpose does not say when getPose() returns null. It probably should never but apparently it does. Is this.xrReferenceSpace null or undefined when this happens ?

Reply:
@andreasplesch 
If i recall correctly, either inputPose or inputPose.transform is null when the VR mode is loaded when controllers are not in the zone covered by sensors. 
https://immersive-web.github.io/webxr/#populate-the-pose mentions when pose can be null

Reply:
Ah, there it is. Ok.

Presumably, if inputPose exists, eg. is not null, it is always an object ?
Similarly, if inputPose.transform exists, eg. is not null, it is always an object ?

Reply:
Yes, the docs mention getPose return an XRPose object and transform is of type XRRigidTransform.

Reply:
So the first, typeof guard should not be needed ?

Reply:
Alright, I have removed typeof checks.